src/contour/CMakeLists.txt

```diff
@@ -285,14 +285,17 @@ else()
         OUTPUT_STRIP_TRAILING_WHITESPACE
     )
 
-    set(YAML_CPP_PKG_NAME "libyaml-cpp0.7")
+    set(DEPENDS_EXTRA ", libyaml-cpp0.7")
     if ("${LSB_RELEASE_NUMBER}" STREQUAL "18.04")
-        set(YAML_CPP_PKG_NAME "libyaml-cpp0.5v5")
+        set(DEPENDS_EXTRA ", libyaml-cpp0.5v5")
+        set(DEPENDS_EXTRA "${DEPENDS_EXTRA}, libboost-filesystem1.65.1")
+    elseif("${LSB_RELEASE_NUMBER}" STREQUAL "20.04")
+        set(DEPENDS_EXTRA ", libyaml-cpp0.6")
     endif()
 
     # Generator: Debian
     # see https://cmake.org/cmake/help/v3.6/module/CPackDeb.html#variable:CPACK_DEBIAN_PACKAGE_DEPENDS
-    set(CPACK_DEBIAN_PACKAGE_DEPENDS "libfreetype6, libharfbuzz0b, libqt5core5a, libqt5gui5, libqt5network5, ${YAML_CPP_PKG_NAME}")
+    set(CPACK_DEBIAN_PACKAGE_DEPENDS "libfreetype6, libharfbuzz0b, libqt5core5a, libqt5gui5, libqt5network5, libqt5multimedia5, libqt5x11extras5${DEPENDS_EXTRA}")
```

Comment:
missing space / typo here?

Reply:
It's not missing. the `${DEPENDS_EXTRA}` is either an empty string or something like `", a"` or `", a, b"`